README.md

```diff
@@ -53,6 +53,10 @@ A React library is available to install under the name `iconoir-react`. For more
 
 A React Native library is available to install under the name `iconoir-react-native`. For more details, see the package [README](./packages/iconoir-react-native).
 
+## Vue
+
+A Vue library is available to install under the name `iconoir-vue`. For more details, see the package [README](./packages/iconoir-vue).
```

Comment:
@lucaburgio / @paescuj should confirm but I believe we decided whenever any new packages are to be released, we'll have them named with an organization prefix.

Can you rename any mention of `iconoir-vue` with `@iconoir/vue` instead?

Reply:
Oh yes, please 😍